config/environments/production.rb

```diff
@@ -22,8 +22,6 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true, mangle: false)
```

Comment:
I think we still want to keep this in production with a version check, until we are officially on Rails 6.